.gitignore

```diff
@@ -15,6 +15,9 @@
 # emacs stuff
 *~
 
+# Ignore the bazel remote cache key file
```

Comment:
Add a link to https://docs.bazel.build/versions/master/remote-caching.html#google-cloud-storage ?

Reply:
Added.